### PR TITLE
User/xianz/winmldb run

### PR DESCRIPTION
**Pull request by zhangxiang1993**

### CODE REVIEW

File: Tools/WinMLDashboard/public/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,5 @@
 git+https://github.com/apple/coremltools
---extra-index-url http://aipkgs/repository/aiinfra/simple --trusted-host aipkgs 
-libsvm==3.22
+lightgbm==2.2.1
 winmltools==1.2.0.912
 tensorflow==1.11.0
```
Comment (kumraj) [inline Tools/WinMLDashboard/public/requirements.txt]: Why is tensorflow needed?

File: Tools/WinMLDashboard/tsconfig.json [hunk 2]
```diff
@@ -26,6 +26,7 @@
     "jest",
     "node_modules",
     "public",
+    "release",
```
Comment (kumraj) [inline Tools/WinMLDashboard/tsconfig.json]: what is this change?

File: Tools/WinMLDashboard/src/view/run/View.tsx [hunk 3]
```diff
@@ -264,25 +267,33 @@ class RunView extends React.Component<IComponentProperties, IComponentState> {
...
+            runDialogOptions.message = 'run failed!'
+            require('electron').remote.dialog.showMessageBox(runDialogOptions)
             return;
         }
         this.setState({
             currentStep: Step.Success,
         });
-        // tslint:disable-next-line:no-console
-        console.log(this.state.currentStep.toString());
+        runDialogOptions.message = 'run successfully';
```
Comment (kumraj) [inline Tools/WinMLDashboard/src/view/run/View.tsx]: should be "Run completed successfully!"

File: Tools/WinMLDashboard/src/view/run/View.tsx [hunk 4]
```diff
@@ -264,25 +267,33 @@ class RunView extends React.Component<IComponentProperties, IComponentState> {
...
+        }
         try {
             await execFilePromise(modelRunnerPath, this.state.parameters, {}, this.outputListener);
         } catch (e) {
+            log.info(this.state.model + " is failed to run");
             this.printError(e);
             this.setState({
                 currentStep: Step.Idle,
             });
+            runDialogOptions.message = 'run failed!'
```
Comment (kumraj) [inline Tools/WinMLDashboard/src/view/run/View.tsx]: Say "Run failed! See console log for details."

File: Tools/WinMLDashboard/src/view/run/View.tsx [hunk 5]
```diff
@@ -94,6 +97,12 @@ class RunView extends React.Component<IComponentProperties, IComponentState> {
     }
 
     private getView = () => {
+        const osInfo = require('os').release()
+        log.info(osInfo);
+        if(osInfo < '10.0.17669') {
```
Comment (kumraj) [inline Tools/WinMLDashboard/src/view/run/View.tsx]: 10.0.17763

File: Tools/WinMLDashboard/src/view/run/View.tsx [hunk 6]
```diff
@@ -94,6 +97,12 @@ class RunView extends React.Component<IComponentProperties, IComponentState> {
...
+            const message = 'This functionality can only be used for R5-later operating system'
```
Comment (kumraj) [inline Tools/WinMLDashboard/src/view/run/View.tsx]: "This functionality is available on Windows 10 October 2018 Update (1809) or newer version of OS."

File: Tools/WinMLDashboard/public/requirements.txt (hunk 1 above)
Comment (zhangxiang1993) [reply Tools/WinMLDashboard/public/requirements.txt]: Tensorflow is needed for Keras. Keras can use tensorflow as backend.

File: Tools/WinMLDashboard/tsconfig.json (hunk 2 above)
Comment (zhangxiang1993) [reply Tools/WinMLDashboard/tsconfig.json]: For locally use, removed.

**Pull request merged**